lib/bob_ross/backends/imagemagick.rb

```diff
@@ -88,6 +92,11 @@ def transform(image, transformations, options)
         }))
       end
     rescue => e
+      @logger.debug <<~DEBUG
+        Exit status is: #{command.exit_status}
+        Input file #{image.source.path} exists? #{File.exist? image.source.path}
+        Output file #{output.path} exists? #{File.exist? output.path}
+      DEBUG
       output.close!
```

Comment:
I would have moved this to an `ensure` block, but I wasn't sure if it would change the program. On second though, it should be fairly safe, but what do you think?